### PR TITLE
User can create an event

### PR DESCRIPTION
**Pull request by hsu-isaac**
final-project-issue-1

### CODE REVIEW

**Comment by hsu-isaac** [comment]
final-project-issue-1 2

File: client/components/form.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              <textarea className="width-100 rounded" name="description" rows="10" id="description" value={description} onChange={handleChange} required></textarea>
+              <h2>Picture</h2>
+              <input type="file" name="image" id="imageUrl" accept="image/png, image/jpeg" value={imageUrl} onChange={handleChange} required></input>
+            </div>
+          </div>
+          <div className="justify-center">
+            <button className="big-button justify-center" type="submit">Create Event</button>
+          </div>
+        </form>
+        <div className="footerSpace"></div>
```
Comment (thebearingedge) [inline client/components/form.jsx]: Try adding margin-bottom to your form instead.

File: client/components/form.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      </>
```
Comment (thebearingedge) [inline client/components/form.jsx]: Then you can lose the fragment.

File: client/pages/home.jsx [hunk 3]
```diff
@@ -1,8 +1,16 @@
 import React from 'react';
+import Navbar from '../components/navbar.jsx';
+import EventForm from '../components/form.jsx';
 
 export default function Home(props) {
   return (
     <>
```
Comment (thebearingedge) [inline client/pages/home.jsx]: This fragment is extraneous

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Pull request merged**